src/System.Net.HttpListener/tests/WebSocketTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Http;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Tests
+{
+    public class WebSocketTests
+    {
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
+        public static async Task AcceptWebSocketAsync_NullSubProtocol_Succeeds()
```

Comment:
We should also look at using HttpListener now as a localhost server for many of our ClientWebSockets tests.  That serves two purposes: 1) avoids us needing to go across the network in a bunch of our tests, making them more reliable, and 2) adds extra testing to HttpListener.  Regardless, thanks for adding at least this one test; we obviously need more after this, even if we do use HttpListener for the ClientWebSockets tests.